packages/bodiless-organisms/src/components/Menu/SimpleMenu.tsx

```diff
@@ -42,6 +42,18 @@ const asMenuSubList = flow(
   withDeleteNodeOnUnwrap,
 );
 
+/**
```

Comment:
Why are we putting this back?

Reply:
At the time it was just moved few lines above unchanged. Git diff made it look like it was deleted and added back.